lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -1,7 +1,7 @@
 const React = require('react');
 
 const title = "Upload photos of yourself in action!";
-const bodies = ["DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"];
+const content = ["DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"];
```

Comment:
Sorry. Forgot to mention this. Mind reformatting when it's an array so it's more obvious what the paragraphs are? One per line.
```javascript
const content = [
  "DoSomething.org staff reviews every single photo you send, and we’ll feature our favorites in the gallery on the site.", 
  "We are PUMPED to see your photos. Complete these simple steps of yourself in action. Your pics will inspire others to join the movement -- upload photos and watch the likes roll in!"
];
```